.github/workflows/build.yml

```diff
@@ -46,67 +32,24 @@ jobs:
       product-version-pre: ${{ steps.get-product-version.outputs.product-version-pre }}
       experiments: ${{ steps.get-product-version.outputs.experiments }}
       go-ldflags: ${{ steps.get-product-version.outputs.go-ldflags }}
+      pkg-name: ${{ steps.get-pkg-name.outputs.pkg-name }}
 
     steps:
       - uses: actions/checkout@v3
       - name: Git Describe
         id: git-describe
+        run: .github/scripts/fetch_raw_version.sh
```

Comment:
Moved in-line shell scripting to separate scripts to clean up file and so bats testing can be added if needed.